hyperspy/tests/io/test_hdf5.py

```diff
@@ -371,6 +371,7 @@ def teardown_method(self, method):
 
 class Test_permanent_markers_io:
 
+    @pytest.mark.skipif("sys.platform == 'darwin'")
```

Comment:
I guess this should be done for all the current plotting tests? If so, I think that belongs in a separate pull request.